### PR TITLE
Feature/Update DB v0.3 and API v1.4

### PR DESCRIPTION
**Pull request by quanpython**

### CODE REVIEW

File: studio/app/optinist/routers/expdb.py [hunk 1]
```diff
@@ -161,6 +177,9 @@ async def search_public_cells(
         additional_data={"header": ExpDbExperimentHeader(graph_titles=graph_titles)},
     )
     for item in data.items:
+        item.experiment_id = (
+            db.query(optinist_model.Experiment).get(item.experiment_uid).experiment_id
```
Comment (itutu-tienday) [inline studio/app/optinist/routers/expdb.py:181]: Are you not issuing a lot of SQL here?
→ Couldn't you handle this with a JOIN?

File: studio/app/optinist/routers/expdb.py [hunk 2]
```diff
@@ -304,7 +333,9 @@ async def search_db_cells(
 
     for item in data.items:
         # TODO: set experiment.id
-        # item.exp_id = item.experiment.id
+        item.experiment_id = (
+            db.query(optinist_model.Experiment).get(item.experiment_uid).experiment_id
```
Comment (itutu-tienday) [inline studio/app/optinist/routers/expdb.py:337]: Are you not issuing a lot of SQL here?
→ Couldn't you handle this with a JOIN?

**Review by itutu-tienday — CHANGES_REQUESTED** [review]
Some remarks are noted.

File: studio/app/optinist/routers/expdb.py (hunk 2 above)
Comment (quanpython) [reply studio/app/optinist/routers/expdb.py]: @itutu-tienday  I implemented. Please check it

**Review by itutu-tienday — COMMENTED** [review]
@quanpython 
The points raised have been addressed and I think the PR is OK.
→ If there are no other issues, it would be good if we can merge them into the branch (#82) on the FrontEnd side. What do you think?

**Comment by quanpython** [comment]
@itutu-tienday 
Yes, I agree with you. 
If there are no other issues. I will change the status of PR

**Review by itutu-tienday — APPROVED** [review]

**Comment by itutu-tienday** [comment]
@quanpython 
> If there are no other issues. I will change the status of PR

Yes, please update PR and issue status.

**Review by ReiHashimoto — APPROVED** [review]

**Pull request merged**